cves/CVE-2016-7122.yml

```diff
@@ -129,7 +143,14 @@ lifetime:
           git log --stat abc..def -- your/file.c
           (where a)
       * the changelog between the two
-  answer:
+  answer: |
+    The lifetime of the vulnerability from its bug discovery to its earliest fix
+    spanned under five years. The bug report was closed within the same year that
+    it was opened and the earliest fix is still the latest fix. The vulnerability
+    itself did not halt the progression of the project or cause any massive 
+    changes. The file had many developers working on over the years with one main 
```

Comment:
I think saying an exact amount of developers is good here to show that you actually looked at the changelog

Reply:
added the exact numbers of contributors to the file